src/pudl/convert/epacems_to_parquet.py

```diff
@@ -125,7 +131,14 @@ def main():
                         else "",
                     },
                 },
-            }
+            },
+            "ops": {
+                "hourly_emissions_epacems": {
+                    "config": {
+                        "partition": args.partition,
+                    }
+                }
+            },
```

Comment:
We should add the --partition flag to the nightly build scripts.

Reply:
True. Since we're no longer using the `epacems_to_parquet` script in the nightly builds I added an option to the etl CLI.

Reply:
Oh smart!